File: src/SistemaComercial.Domain/Entities/FormaPago.cs
using System.Text.Json.Serialization;

namespace SistemaComercial.Domain.Entities;

public class FormaPago
{
    public short IdFormaPago { get; set; }

    public string Nombre { get; set; } = string.Empty;

    public bool Estado { get; set; } = true;

    // Navegación
    [JsonIgnore]
    public ICollection<DetallePago> DetallesPago { get; set; } = [];
}

File: frontend/src/api/formaPago.ts
import { api } from "./axios";

export interface FormaPago {

    idFormaPago: number;

    nombre: string;

    estado: boolean;

}

export async function obtenerFormasPago() {

    const response = await api.get<FormaPago[]>(
        "/FormaPago",
    );

    return response.data;

}